nautobot_netbox_importer/diffsync/models/dcim.py

```diff
@@ -54,26 +54,15 @@ def setup(adapter: SourceAdapter) -> None:
             "role": fields.role(adapter, "dcim.rackrole"),
         },
     )
-    adapter.configure_model(
-        "dcim.cable",
-        fields={
-            "termination_a": "termination_a",
-            "termination_b": "termination_b",
-        },
-    )
+    adapter.configure_model("dcim.cable")
     adapter.configure_model(
         "dcim.cabletermination",
         extend_content_type="dcim.cable",
         pre_import=_pre_import_cable_termination,
-        fields={
-            "termination_a": "termination_a",
-            "termination_b": "termination_b",
-        },
     )
     adapter.configure_model(
         "dcim.interface",
         fields={
-            "status": "status",
```

Comment:
why status is not needed anymore?

Reply:
Refactoring allows auto-adding fields that are crucial, as `status` is.

https://github.com/nautobot/nautobot-app-netbox-importer/pull/123/files#diff-6b479358489e04d98598db52da0a5cce8c2636d21202d851e16589d38820e84eR102

https://github.com/nautobot/nautobot-app-netbox-importer/pull/123/files#diff-fe77f230ddce96ba9a9d7b349e5e7acfe3c00818cd1818c543d2c1f29504d39dR412